src/tests/install_update_test.ts

```diff
@@ -0,0 +1,223 @@
+import { assertEquals } from "../dev_deps.ts";
+import { mockFile } from "../dev_deps.ts";
+import {
+  createUpdateResp,
+  SDK_NAME,
+  updateDependencyFile,
+  updateDependencyMap,
+} from "../install-update.ts";
+
+const MOCK_RELEASES = [
+  {
+    name: "deno_slack_sdk",
+    current: "0.0.6",
+    latest: "0.0.7",
+    update: true,
+    breaking: false,
+    error: null,
+  },
+  {
+    name: "deno_slack_api",
+    current: "0.0.6",
+    latest: "0.0.7",
+    update: true,
+    breaking: false,
+    error: null,
+  },
+  {
+    name: "deno_slack_hooks",
+    current: "0.0.9",
+    latest: "0.0.10",
+    update: true,
+    breaking: false,
+    error: null,
+  },
+];
+
+const MOCK_SLACK_JSON = JSON.stringify({
+  hooks: {
+    "get-hooks":
+      "deno run -q --allow-read --allow-net https://deno.land/x/deno_slack_hooks@0.0.9/mod.ts",
+  },
+});
+
+const MOCK_IMPORT_MAP_JSON = JSON.stringify({
+  imports: {
+    "deno-slack-sdk/": "https://deno.land/x/deno_slack_sdk@0.0.6/",
+    "deno-slack-api/": "https://deno.land/x/deno_slack_api@0.0.6/",
+  },
+});
+
+const MOCK_SLACK_JSON_FILE = new TextEncoder().encode(MOCK_SLACK_JSON);
+const MOCK_IMPORT_MAP_FILE = new TextEncoder().encode(MOCK_IMPORT_MAP_JSON);
+
+Deno.test("update hook tests", async (t) => {
+  await t.step("createUpdateResp", async (evT) => {
+    await evT.step(
+      "if import_map.json has available updates, then response includes those updates",
+      async () => {
+        mockFile.prepareVirtualFile(
+          "./slack.json",
+          MOCK_SLACK_JSON_FILE,
+        );
+
+        mockFile.prepareVirtualFile("./import_map.json", MOCK_IMPORT_MAP_FILE);
+        mockFile.prepareVirtualFile("./slack.json", MOCK_SLACK_JSON_FILE);
```

Comment:
How come we are preparing the slack.json file twice in this test?

Reply:
It was getting wild in here for a bit testing-wise. Remnant of that. Will remove!